src/main/java/org/jabref/gui/fieldeditors/FieldEditors.java

```diff
@@ -25,16 +28,40 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.jabref.model.entry.FieldName.AUTHOR;
+import static org.jabref.model.entry.FieldName.INSTITUTION;
+import static org.jabref.model.entry.FieldName.TITLE;
+import static org.jabref.model.entry.FieldName.YEAR;
+
 public class FieldEditors {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(FieldEditors.class);
 
-    public static FieldEditorFX getForField(String fieldName, TaskExecutor taskExecutor, DialogService dialogService, JournalAbbreviationLoader journalAbbreviationLoader, JournalAbbreviationPreferences journalAbbreviationPreferences, JabRefPreferences preferences, BibDatabaseContext databaseContext, String entryType, SuggestionProviders suggestionProviders, UndoManager undoManager) {
+    private static final Set<String> SINGLE_LINE_FIELDS = Collections.unmodifiableSet(new HashSet<>(
+            Arrays.asList(TITLE, AUTHOR, YEAR, INSTITUTION)
+    ));
+
+    public static FieldEditorFX getForField(final String fieldName,
+                                            final TaskExecutor taskExecutor,
+                                            final DialogService dialogService,
+                                            final JournalAbbreviationLoader journalAbbreviationLoader,
```

Comment:
You could extract the loader and the prefs parameter and directly pass the  JounralAbbrev Repo stuff which is just passed onto the field checkers later 
`journalAbbreviationLoader.getRepository(journalAbbreviationPreferences)`

Reply:
done

Reply:
In principle, the user might change the journal abbreviations at any time. However, it should be fine to use the repository since the field editors are recreated when switching to the new entry.

Reply:
In principle, the user might change the journal abbreviations at any time. However, it should be fine to use the repository since the field editors are recreated when switching to the new entry.